### ISSUE TITLE
Feature request: INSET shadow

### ISSUE THREAD

**Issue by TomasSestak** [SrBrahma/react-native-shadow-2#10]

**Comment by SrBrahma** [comment]
I am struggling to fix a final bug in the next major version. When released, I may start working on that. Meanwhile, if you are not using Expo, you may want to take a look at [react-native-neomorph-shadows](https://github.com/tokkozhin/react-native-neomorph-shadows)

**Comment by lukas-sl** [comment]
Are there any updates on this? I'm looking for a package that supports inset shadows (like [react-native-neomorph-shadows](https://github.com/tokkozhin/react-native-neomorph-shadows)) but since my project is managed by Expo, I can't use it. Do you have any plans on implementing inset shadows in the near future?

**Comment by SrBrahma** [comment]
Hi, @lukas-sl!

I tried back then to make the inset shadow. While the basics worked fine, I had some problematic cases that I couldn't work around. As no one else requested it (and I never needed it), it wouldn't worth the effort to try harder.

What is your use case? Can you provide me an image example, maybe in html/css and its code?

I will soon have more free time and I can try to get it working, and release it together with the upcoming 7.0.0 version.

**Comment by lukas-sl** [comment]
Hi @SrBrahma,

thanks for your reply and sorry for my late answer. You can see an example of my use case on https://neumorphism.io/ if you select the "Pressed" variant under the Shape option. 

If it's a lot of work to get it working properly, don't bother with it. We just upgraded our project to the latest Expo version which comes with React Native 0.68. This allows us to use https://github.com/Shopify/react-native-skia which provides inset shadows as well.

**Comment by SrBrahma** [comment]
Thanks!

I am aware of Skia! Before they made their shadow, I was considering to port this lib to use the skia instead of svg, until I saw it didn't support Expo yet. Yup! Looks like their shadow will at some point be more popular than this lib, it's a better technology than SVG with all my hacks to make the shadow.

I haven't used it yet. How their shadow looks like? Can it wraps general components and does it works on the first render without entering the width and height? Does it support what my lib does? No problem if it does and if it's better hehe! I was actually already planning to write about Skia in the 7.0.0 release text, on how certainly the RN's shadow future is by using it.

Anyway, as I got some free time these days, I will try to get the `inset` done, along with others pendency.
